lib/instrumentation/index.js

```diff
@@ -379,8 +375,6 @@ Instrumentation.prototype.bindFunction = function (original) {
     ins.currentTransaction = trans
     ins.bindingSpan = null
     ins.activeSpan = span
-    if (trans) trans.sync = false
-    if (span) span.sync = false
```

Comment:
Question: Am I correct that a consequences of removing "`if (span) span.sync = false`" is that folks using the patch-async nodejs-context tracker (i.e. [the `asyncHooks` configuration ](https://www.elastic.co/guide/en/apm/agent/nodejs/master/configuration.html#async-hooks) is set to false) will no longer have any tracking of `span.sync` value?  If so, and if we're comfortable with this trade off (I am)  this seems like it's worth mentioning in the configuration docs for the `asyncHooks` configuration and/or calling out with a CHANGELOG entry.

Reply:
Actually, even with `asyncHooks=false` this still works. `span.sync` determination is now totally separate from what context manager is being used (patch-async, or async-hooks ... and eventually AsyncLocalStorage-based, etc.). The only requirement is that `async_hooks.executionAsyncId()` is available -- which it is since Node v8.2.0.

Reply:
Ah -- it's not reliant our hook specifically 👍